site/_docs/history.md

```diff
@@ -28,10 +28,21 @@ For a full list of releases, see
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.26.0">1.26.0</a> / 2020-xx-xx
+{: #v1-26-0}
+
+Under development.
+
+Compatibility: This release is tested on Linux, MacOS, Microsoft Windows;
+using Oracle JDK 8, 9, 10, 11, 12, 13, 14, 15
+and OpenJDK 8, 9, 10, 11, 12, 13, 14, 15;
+Guava versions 19.0 to 29.0-jre;
```

Comment:
Where are we testing all this? I am asking out of curiosity since GitHub and Apache Jenkins CI does not seem to cover all these options.

Reply:
Frankly speaking, I would exclude known-outdated versions from the list.
Who cares if Calcite works with Java 9 and 10? The versions are obsolete long ago.
Who cares if Calcite works with Java 12 and 14? No key Java vendors list 12 and 14 as long or mid-term supported versions.

As a committer, I assume the code is good if GitHub CI / Travis CI checks pass. I would never even try to compile Calcite with Java 9, 10, 12, and 14. I assume every commit might break compatibility with 9, 10, 12, and 14.

---

>and OpenJDK 8, 9, 10, 11, 12, 13, 14, 15;

There's no such thing as OpenJDK distribution. So we might want to re-phrase this `Oracle JDK / OpenJDK`.
Do we want to mention platforms and architectures as well?